### ISSUE TITLE
fix failing travis.ci 

### ISSUE THREAD

**Issue by 7yl4r** [EmergentOrganization/cell-rpg#114]
travis.ci builds are failing now though I all I changed was the readme. Our travis.yaml is probably out-of-date or something I guess.

**Comment by orelb** [comment]
I will give it a look when I get home, but I barely know anything about CI.

**Comment by orelb** [comment]
I think Travis is really valuable to the project but there's some stuff that bothers me about our Travis build.

We are starting up an Android emulator and running the desktop tests on him. That's reasonable, what works on desktop won't necessarily run on Android. 
However, the way we create the AVD (Android Virtual Device) using the [android](https://developer.android.com/studio/tools/help/android.html) command line tool is not supported.

> This tool is no longer supported. Instead use Android Studio to create AVDs and create projects, or use sdkmanager to view and install SDK packages from the command line.

Plenty of people still use this way (this is the only interface I could find), and this is how the Travis CI [wiki](https://docs.travis-ci.com/user/languages/android/#How-to-Create-and-Start-an-Emulator) does it. So I'm not sure if there's an alternative.

I was just researching some of the SDK and came across this issue, not sure what we could do about it really. This is probably the only way to automate the test on Travis.

On a side note: we should probably move the CA tests to the core project, and when building android or desktop they'll run those tests because they share them

**Comment by 7yl4r** [comment]
Yeah, our travis setup is definitely not ideal. I think I went with whatever I got working most quickly.
